components/Skeleton/Skeleton.tsx

```diff
@@ -1,9 +1,20 @@
-interface SkeletonProps {
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import { JSX } from "react";
+
+interface ISkeletonProps {
```

Comment:
refactor (potentially blocking): @shashilo, are we using interface names prefixed with `I`?

```suggestion
interface SkeletonProps {
```

Reply:
@nickytonline I was told interfaces are prefaced with `I`